.github/workflows/e2e-tests-reviews.yml

```diff
@@ -22,11 +22,9 @@ jobs:
     steps:
       - uses: actions/setup-node@v3.5.1
       - uses: actions/checkout@v4
-      # - uses: fregante/setup-git-user@v2
         with:
           repository: gctools-outilsgc/gcpedia
           token: ${{ secrets.BOT_ACCESS_TOKEN }}
```

Comment:
I'm not sure if this or the other ones are actually required for intended behaviour, but they are technically valid for `checkout`